.github/workflows/verify-prs.yml

```diff
@@ -0,0 +1,31 @@
+name: Verify PRs
```

Comment:
@solemabrothers Why verify PRs why not Run Tests?

Reply:
Done